data/operators/power/line.json

```diff
@@ -1653,33 +1641,6 @@
         "power": "line"
       }
     },
-    {
-      "displayName": "EnBW",
-      "id": "enbw-d5e323",
-      "locationSet": {"include": ["de"]},
-      "matchNames": [
-        "enbw ag",
```

Comment:
`enbw ag` should either be excluded or included in a matchNames of another entry

Reply:
I excluded them, as the new operator could be both either Netze BW or TransnetBW.